packages/cli/src/databases/mysqldb/migrations/1636626154933-CreateUserManagement.ts

```diff
@@ -109,10 +109,6 @@ export class CreateUserManagement1636626154933 implements MigrationInterface {
 			`ALTER TABLE ${tablePrefix}workflow_entity DROP INDEX IDX_${tablePrefix}943d8f922be094eb507cb9a7f9`,
 		);
 
-		await queryRunner.query(
```

Comment:
I believe this file is missing the `logMigrationStart` or are we adding this to a few files specifically?

Reply:
I just added it to SQLite for now, since MySQL and Postgres run migrations in transactions and thus don't break anything on interrupt.